.gitignore

```diff
@@ -3,3 +3,4 @@ pom.xml.tag
 pom.xml.releaseBackup
 pom.xml.next
 release.properties
+nb-configuration.xml
```

Comment:
Please put IDE-specific files that you want ignored by version control into ~/.gitignore_global instead of in the project's .gitignore.